Formula/d/debianutils.rb

```diff
@@ -23,7 +23,14 @@ class Debianutils < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "6fd96fb4733f4fc575514e778d83ee69006440d867df416ed58b45bfb954ae30"
   end
 
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "po4a" => :build
+  depends_on "gettext" # for libintl
```

Comment:
Is `gettext` a build dependency? It doesn't make sense for those utilities (`tempfile`, `ischroot`, `run-parts`) to depend on `gettext`.